src/main/java/glowredman/amazingtrophies/AmazingTrophies.java

```diff
@@ -52,7 +52,7 @@ public class AmazingTrophies {
     public static final String VERSION = Reference.VERSION;
     public static final Logger LOGGER = LogManager.getLogger(MODNAME);
     public static final Path CONFIG_DIR = getConfigDir();
-    public static boolean isAngelicaLoaded = false;
+    public static final boolean enableVBO = !Boolean.parseBoolean(System.getProperty("amazingtrophies.disableVBO", "false"));
```

Comment:
Here you could use `Boolean.getBoolean("amazingtrophies.disableVBO")` instead.

Reply:
ah nice, wasn't aware of that